### DISCUSSION TITLE
confused with identical model names

### DISCUSSION THREAD

**Discussion by wstlxx** [open-webui/open-webui#5423] [General]
**Is your feature request related to a problem? Please describe.**
 I'm always frustrated when having models from different providers, and come across several identical models.
they come from different providers but have same model names from the home page.

**Describe the solution you'd like**
I would prefer a easy on-off switch for each api-key when i don't want some slow but cheap provider show up.
Or, we could add some hashtag after model name with alias written within settings.

**Describe alternatives you've considered**
or maybe we can filter provider from the home page without having to navigate into settings

**Additional context**
BTW, we cannot  delete the 1st key in the settings while I simply want to turn it off. I have to delete the 2nd key, and change the content of 1st key into that 2nd one, in order to use it exclusively.

**Comment by MRGRD56** [comment]
As a workaround, you can make unwanted provider urls invalid, for example, by adding `-` before them, so they won't be used.

![image](https://github.com/user-attachments/assets/c0b2e669-113d-4318-8850-25457dd71c71)
